packages/frontpage/app/api/receive_hook/handlers.ts

```diff
@@ -109,22 +116,24 @@ export async function handleComment({ op, repo, rkey }: HandlerInput) {
       });
     } else {
       const { content, createdAt, parent, post } = commentRecord.value;
+      const postUri = AtUri.parse(post.uri);
+      const parentUri = parent ? AtUri.parse(parent.uri) : null;
```

Comment:
I don't really like this tbh, I'd like if the generated client would parse to `AtUri` for us. Raised in #3767